pom.xml

```diff
@@ -207,76 +221,6 @@
-                        <artifactId>maven-assembly-plugin</artifactId>
-                        <version>3.6.0</version>
-                        <configuration>
-                            <appendAssemblyId>false</appendAssemblyId>
-                            <descriptors>
-                                <descriptor>src/main/assembly/assembly.xml</descriptor>
-                            </descriptors>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <goals>
-                                    <goal>single</goal>
-                                </goals>
-                                <phase>install</phase>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-source-plugin</artifactId>
-                        <version>3.3.0</version>
-                        <executions>
-                            <execution>
-                                <id>attach-sources</id>
-                                <phase>verify</phase>
-                                <goals>
-                                    <goal>jar-no-fork</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-javadoc-plugin</artifactId>
-                        <version>3.5.0</version>
-                        <executions>
-                            <execution>
-                                <id>attach-javadocs</id>
-                                <phase>verify</phase>
-                                <goals>
-                                    <goal>jar</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                        <configuration>
-                            <quiet>true</quiet>
-                            <doclint>none</doclint>
-                        </configuration>
-                    </plugin>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-jar-plugin</artifactId>
-                        <version>${maven.jar.plugin.version}</version>
-                        <configuration>
-                            <!-- Generated file that shouldn't be included in add-ons -->
-                            <excludes>
-                                <exclude>META-INF/VAADIN/config/flow-build-info.json</exclude>
-                                <exclude>META-INF/VAADIN/webapp/**/*</exclude>
```

Comment:
Ok, we have this part in the build, check line 120/134.
 and line 153 does not contain the `  <exclude>META-INF/VAADIN/webapp/**/*</exclude>` 
 So if no objection then I will not add this line, and have the original `maven-jar-plugin configuration` in place.

Reply:
Currently, form filler add-on has nothing to be placed into `VAADIN/webapp`.
Usually it's a bundle static resources or other static resource provided by Vaadin, that are placed there (e.g. JavaScript for Push).
However, if we place this exclusion and in the future we add some resources into add-on, we might struggle why it doesn't work properly.
So I think we can skip adding it at the moment.

Reply:
Thank you @mshabarov for your insights here and in the daily as well. It makes sense!
Also thanks for approve and merge :)